modules/packages/HDFS.chpl

```diff
@@ -176,7 +182,7 @@ First reflect your directory structure and version numbers (etc) in the
 
 2. Install Hadoop
```

Comment:
This file does not have anything about libhdfs3. According to http://chapel.cray.com/docs/1.13/technotes/auxIO.html#parallel-and-distributed-i-o-features, the file `$CHPL_HOME/third-party/libhdfs3/README` describes the instructions for libhdfs3.